oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/SQLExecutor.java

```diff
@@ -54,4 +54,12 @@ public void invoke(Connection connection) throws SQLException {
         }
         preparedStatement.execute();
     }
+
+    public String getSql() {
+        return sql;
+    }
+
+    public List<Object> getParam() {
+        return param;
+    }
```

Comment:
This should be replaced by https://github.com/apache/skywalking/pull/7691/files#r706947361

Reply:
ok. remove getParam method